### ISSUE TITLE
early stopping , could not train after 10 epoch

### ISSUE THREAD

**Issue by manvirvirk** [4uiiurz1/pytorch-nested-unet#16]
100%|██████████████████████████████████████████████████████████████| 1/1 [00:01<00:00,  1.95s/it, loss=0.671, iou=0.63]
100%|██████████████████████████████████████████████████████████| 1/1 [00:01<00:00,  1.71s/it, loss=0.788, iou=3.97e-10]
loss 0.6712 - iou 0.6299 - val_loss 0.7878 - val_iou 0.0000
=> early stopping

**Comment by 4uiiurz1** [comment]
@manvirvirk plz set `early_stopping=-1`.

**Comment by wxyWendy** [comment]
Do you use deep supervision?
